src/applications/auth/containers/AuthApp.jsx

```diff
@@ -16,7 +20,7 @@ export class AuthApp extends React.Component {
   }
 
   componentDidMount() {
-    if (!this.state.error) this.validateSession();
+    if (!this.state.error || hasSession()) this.validateSession();
```

Comment:
If the session is still active, just proceed with validating it so that the user can get redirected away from this page.

Reply:
This doesn't really affect most cases. The only time I would ever see this come into effect is when a logged-in user somehow directly goes to the callback URL with the error code, which I don't see happening normally.